db/migrate/20180410002829_create_conditions.rb

```diff
@@ -0,0 +1,15 @@
+class CreateConditions < ActiveRecord::Migration[5.1]
+  def change
+    create_table :conditions do |t|
+      t.string :date
+      t.float :max_temperature
+      t.float :mean_temperature
+      t.float :min_temperature
```

Comment:
@hmesander @jrambold @jamesrnelson  Are you guys ok with me leaving off the _f for the max mean and min temperature attributes. I feel like it makes for cleaner code, but it is a change from the CSV headers.

Reply:
That seems fine to me. Let's do it.

Reply:
I'm alright leaving this off as we don't have celsius value anywhere that would cause confusion